core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1603,7 +1602,8 @@ public Label getLabel(String expr) {
      * Returns the label atom of the given name.
      * @return non-null iff name is non-null
      */
-    public @Nullable LabelAtom getLabelAtom(@CheckForNull String name) {
+    @CheckForNull
+    public LabelAtom getLabelAtom(@CheckForNull String name) {
```

Comment:
No, `@Nullable` was correct here. If your code passes in a name known to not be null, you can be assured that the result will not be null either.